src/engine/shared/map.cpp

```diff
@@ -111,7 +111,7 @@ bool CMap::Load(const char *pMapName)
 
 					if(((int)TilemapCount / pTilemap->m_Width != pTilemap->m_Height) || (TilemapSize / sizeof(CTile) != TilemapCount))
 					{
-						log_error("map/load", "map layer too big (%d * %d * %d causes an integer overflow)", pTilemap->m_Width, pTilemap->m_Height, sizeof(CTile));
+						log_error("map/load", "map layer too big (%d * %d * %ld causes an integer overflow)", pTilemap->m_Width, pTilemap->m_Height, sizeof(CTile));
```

Comment:
```suggestion
						log_error("map/load", "map layer too big (%d * %d * %d causes an integer overflow)", pTilemap->m_Width, pTilemap->m_Height, (int)sizeof(CTile));
```

`%ld` is just as incorrect. You need to cast the `sizeof` or use `%zu`.

Reply:
clang suggested it

maybe the CI tooling is too old?